<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicMultiScrapeDialog</class>
 <widget class="QDialog" name="MusicMultiScrapeDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Music Multi Scrape</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../data/MediaElch.qrc">
    <normaloff>:/img/MediaElch.png</normaloff>:/img/MediaElch.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Infos to load</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="MyCheckBox" name="chkBorn">
          <property name="text">
           <string>Born</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="type" stdset="0">
           <string notr="true">artist</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="MyCheckBox" name="chkName">
          <property name="text">
           <string>Name</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="type" stdset="0">
           <string notr="true">artist</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="MyCheckBox" name="chkYearsActive">
          <property name="text">
           <string>Years Active</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="type" stdset="0">
           <string notr="true">artist</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="MyCheckBox" name="chkDied">
          <property name="text">
           <string>Died</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="type" stdset="0">
           <string notr="true">artist</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="MyCheckBox" name="chkFormed">
          <property name="text">
           <string>Formed</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="type" stdset="0">
           <string notr="true">artist</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="MyCheckBox" name="chkDisbanded">
          <property name="text">
           <string>Disbanded</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="type" stdset="0">
           <string notr="true">artist</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="MyCheckBox" name="chkBiography">
          <property name="text">
           <string>Biography</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="type" stdset="0">
           <string notr="true">artist</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="MyCheckBox" name="chkCdArt">
          <property name="text">
           <string>CD Art</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="type" stdset="0">
           <string notr="true">album</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="MyCheckBox" name="chkCover">
          <property name="text">
           <string>Cover</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="type" stdset="0">
           <string notr="true">album</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="MyCheckBox" name="chkLogo">
          <property name="text">
           <string>Logo</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="type" stdset="0">
           <string notr="true">artist</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="MyCheckBox" name="chkExtraFanarts">
          <property name="text">
           <string>Extra Fanarts</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="type" stdset="0">
           <string notr="true">artist</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="MyCheckBox" name="chkFanart">
          <property name="text">
           <string>Fanart</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="type" stdset="0">
           <string notr="true">artist</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="MyCheckBox" name="chkThumbnail">
          <property name="text">
           <string>Thumbnail</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="type" stdset="0">
           <string notr="true">artist</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="MyCheckBox" name="chkMoods">
          <property name="text">
           <string>Moods</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="type" stdset="0">
           <string notr="true">both</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="MyCheckBox" name="chkStyles">
          <property name="text">
           <string>Styles</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="type" stdset="0">
           <string notr="true">both</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="MyCheckBox" name="chkGenres">
          <property name="text">
           <string>Genres</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="type" stdset="0">
           <string notr="true">both</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="MyCheckBox" name="chkLabel">
          <property name="text">
           <string>Label</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="type" stdset="0">
           <string notr="true">album</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="MyCheckBox" name="chkReleaseDate">
          <property name="text">
           <string>Release Date</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="type" stdset="0">
           <string notr="true">album</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="MyCheckBox" name="chkRating">
          <property name="text">
           <string>Rating</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="type" stdset="0">
           <string notr="true">album</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="MyCheckBox" name="chkYear">
          <property name="text">
           <string>Year</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="type" stdset="0">
           <string notr="true">album</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="MyCheckBox" name="chkReview">
          <property name="text">
           <string>Review</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="type" stdset="0">
           <string notr="true">album</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="MyCheckBox" name="chkArtist">
          <property name="text">
           <string>Artist</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="type" stdset="0">
           <string notr="true">album</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="MyCheckBox" name="chkTitle">
          <property name="text">
           <string>Title</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="type" stdset="0">
           <string notr="true">album</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="MyCheckBox" name="chkDiscography">
          <property name="text">
           <string>Discography</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="type" stdset="0">
           <string notr="true">artist</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkUnCheckAll">
        <property name="text">
         <string>(Un)Check all</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>51</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QCheckBox" name="chkScrapeAllAlbums">
     <property name="text">
      <string>Scrape all albums of selected artists (and not only selected albums)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkAutoSave">
     <property name="text">
      <string>Automatically save each artist and album after scraping</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
     <item>
      <widget class="QLabel" name="itemCounter">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string notr="true">1/20</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="itemName">
       <property name="text">
        <string notr="true">Item</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressItem">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressAll">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStartScraping">
       <property name="text">
        <string>Start Scraping</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyCheckBox</class>
   <extends>QCheckBox</extends>
   <header location="global">ui/small_widgets/MyCheckBox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../data/MediaElch.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>MusicMultiScrapeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>463</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>MusicMultiScrapeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>463</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
